### ISSUE TITLE
Chart Repository URL is not remembered when editing in loft

### ISSUE THREAD

**Issue by blairham** [loft-sh/loft#49]
After installing an app from a custom repository, the URL is not remembered when editing the app in loft.

![image](https://user-images.githubusercontent.com/5043780/93370986-9f639680-f81f-11ea-9870-39ad8951414c.png)

**Comment by blairham** [comment]
I'm going to close this, as I understand what happened here. I had manually added these which did not have all data necessary

**Comment by blairham** [comment]
After poking around it seems that it's only with the gloo helm chart, it could be the type of helm chart repo:

https://docs.solo.io/gloo/latest/installation/gateway/kubernetes/

**Comment by LukasGentele** [comment]
@blairham Yes, unfortunately, helm does not store the repo URL in the secret that contains the values and other information the release. There are issues for this in the helm repo on GitHub but it will take some time for this to be merged. When installing or upgrading helm charts via Loft UI, we add the annotation `loft.sh/url: URL` to the secret containing the helm release information, so that the UI can display it and also perform version checks / upgrades etc. So, setting this annotation manually on the helm secret should fix this.

**Comment by blairham** [comment]
@LukasGentele I saw that and played with it for the day, that particular one would not work even when setting that annotation on the secret

here is the yaml output of that secret
```
kind: Secret
metadata:
  annotations:
    loft.sh/url: http://storage.googleapis.com/gloo-ee-helm
  creationTimestamp: "2020-09-16T23:22:17Z"
  labels:
    loft.sh/app: "true"
    modifiedAt: "1600298542"
    name: glooe
    owner: helm
    status: deployed
    version: "1"
  name: sh.helm.release.v1.glooe.v1
  namespace: gloo-system
  resourceVersion: "37027659"
  selfLink: /api/v1/namespaces/gloo-system/secrets/sh.helm.release.v1.glooe.v1
  uid: ce7ce641-523f-4883-90d0-21ceb6dc8e86
type: helm.sh/release.v1
```

**Comment by FabianKramm** [comment]
@blairham yes that is a bug, I'll fix it for the next release.

**Comment by FabianKramm** [comment]
@blairham should be fixed with v1.0.1
